build.go

```diff
@@ -14,11 +14,13 @@ import (
 )
 
 var verbose bool
+var autopointPath string
 var folders = []string{"openssl", "libevent", "zlib", "xz", "tor"}
 var absCurrDir = getAbsCurrDir()
 
 func main() {
 	flag.BoolVar(&verbose, "verbose", false, "Whether to show command output")
+	flag.StringVar(&autopointPath, "autopoint-path", "/usr/local/opt/gettext/bin", "OSX: Directory that contains autopoint binary")
```

Comment:
I think it's better to have this on the PATH, use https://golang.org/pkg/os/exec/#LookPath to check it, and not have a separate CLI arg for it

Reply:
After offline discussion, going to leave this to keep the user from having to litter their PATH